app/src/main/java/com/vero/tasky/auth/presentation/login/LoginViewModel.kt

```diff
@@ -54,23 +53,27 @@ class LoginViewModel @Inject constructor(
         }
     }
 
-    private fun logIn() {
-        EmailErrorHandler.validateEmail(
-            isEmailValid = state.isEmailValid,
-            onEmailIsValid = ::onEmailIsValid,
-            showError = ::showError
-        )
+    private fun validateUserInfo() {
+        if (state.isEmailValid) {
```

Comment:
I personally run the validations separate of each other, so if the email and password both are wrong, I show both errors
Not just email